spk/sickrage/src/wizard/install_uifile

```diff
@@ -0,0 +1,22 @@
+[{
+    "step_title": "Choose your fork",
+    "items": [{
+        "type": "textfield",
+        "desc": "Configure the fork to use",
```

Comment:
Not sure there are a lot of forks out there, maybe add something like "skip this to install default sickrage"